shared-bindings/_pew/PewPew.c

```diff
@@ -88,12 +88,12 @@ STATIC mp_obj_t pewpew_make_new(const mp_obj_type_t *type, size_t n_args,
     mp_obj_get_array(args[ARG_cols].u_obj, &cols_size, &cols);
 
     if (bufinfo.len != rows_size * cols_size) {
-        mp_raise_ValueError(translate(""));
+        mp_raise_ValueError(translate("Incorrect buffer size"));
     }
```

Comment:
@deshipu said he didn't include error messages here because it's not really user code, and he was having space issues. See https://github.com/adafruit/circuitpython/pull/1607#discussion_r261618670 (scroll mid-page to see the discussion). Can you still build pewpew with the messages?

Reply:
I can build pewpew ok. The german build has ~2k free. I added it to .travis.yml so that is validated.

I added the messages because gettext is unhappy with empty translations. I think it improves their clarity as well for when they do occur. If they are fixed at build time then asserts could be used instead of Python exceptions.

Reply:
It looks like there is so much free space freed in 4.x, that there are no space issues anymore, and I even could enable additional modules, so there shouldn't be any problems with those.